tests/test_2306_cppyy_git.py

```diff
@@ -114,5 +114,5 @@ def test_array_as_templated_type():
 
     cppyy.cppdef(source_code_cpp)
 
-    out = cppyy.gbl.go_fast_cpp_2[array.cpptype](array)
+    out = cppyy.gbl.go_fast_cpp_2[array.cpp_type](array)
```

Comment:
@agoose77 - local tests pass!

Reply:
@ianna thanks! I built a wheel last night but I didn't think I needed to build the whole stack, and apparently that was a mistake! I was going to re-run it this morning, but you've saved me the trouble!